frontend/src/common/constants/default-error.constants.ts

```diff
@@ -0,0 +1,4 @@
+const DEFAULT_ERROR_TITLE = 'Error';
+const DEFAULT_ERROR_MESSAGE = 'Unexpected error';
```

Comment:
actually it isn't an reusable think
so declarate it in the `frontend/src/services/notification/notification.service.tsx`